spec/models/blueprint_spec.rb

```diff
@@ -34,6 +34,10 @@
     FactoryGirl.create(:service_template, :name => 'Service Template Bundle', :display => true, :blueprint => subject)
   end
 
+  it 'is taggable' do
+    expect(subject).to respond_to(:tag_with)
```

Comment:
Is there a way you could write this to test the behavior, rather than the implementation?